core/primitives-core/src/config.rs

```diff
@@ -290,6 +291,31 @@ pub struct ExtCostsConfig {
     pub validator_stake_base: Gas,
     /// Cost of calling `validator_total_stake`.
     pub validator_total_stake_base: Gas,
+
+    // #############
+    // # Alt BN128 #
+    // #############
+    /// Base cost for multiexp
+    #[cfg(feature = "protocol_feature_alt_bn128")]
```

Comment:
This could be addressed in a refactor PR, but @matklad might know a way we can wrap all of these fields additions with just one flag instead of each.

Reply:
Why not just have an inner struct under feature?

Reply:
Yeah, given that all these start with `alt_bn128_g1_`, it really wants to be

```rust
pub alt_bn128_g1: AltBn128g1
```

regardless of a feature flags.

Reply:
This might break cost counter right now

Reply:
Discussed that we'll refactor it later